src/Facebook/PseudoRandomString/McryptPseudoRandomStringGenerator.php

```diff
@@ -27,44 +27,42 @@
 
 class McryptPseudoRandomStringGenerator implements PseudoRandomStringGeneratorInterface
 {
+    use PseudoRandomStringGeneratorTrait;
 
-  use PseudoRandomStringGeneratorTrait;
+    /**
+     * @const string The error message when generating the string fails.
+     */
+    const ERROR_MESSAGE = 'Unable to generate a cryptographically secure '
+        .'pseudo-random string from mcrypt_create_iv(). ';
```

Comment:
Seeing as how there is no *hard* limit on lines, it is just a single string, and I enjoy playing nice with others.